src/reducers/content.js

```diff
@@ -27,15 +29,23 @@ export default (state = initialState, action) => {
     case CONTENT_ADD_CHANGE:
       return {
         ...state,
-        contentAddByBundle: {
-          ...state.contentAddByBundle,
+        restorableContentAddByBundle: {
+          ...state.restorableContentAddByBundle,
           [action.payload.bundle]: action.payload.entity,
         },
       };
+    case CONTENT_EDIT_CHANGE:
+      return {
+        ...state,
+        restorableContentEditById: {
```

Comment:
I'm wondering whether storing every editing content over time is the right approach. Maybe we should limit the amount of stored entires to let's say the last three?

Reply:
Let's open an issue and figure out the correct logic for that. I think for now this is perfectly fine.

Reply:
👍 Let's be pragmatic indeed

Reply:
Here is one: https://github.com/jsdrupal/drupal-admin-ui/issues/432